### PR TITLE
DOC-4383 added Boost.redis to community client table

### PR DESCRIPTION
**Pull request by andy-stark-redis**
[DOC-4383](https://redislabs.atlassian.net/browse/DOC-4383)

[DOC-4383]: https://redislabs.atlassian.net/browse/DOC-4383?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: content/develop/connect/clients/_index.md [hunk 1]
```diff
@@ -39,10 +39,11 @@ Redis does not support directly:
 | Language | Client name | Github | Docs |
 | :-- | :-- | :-- | :-- |
 | C | hiredis | https://github.com/redis/hiredis | https://github.com/redis/hiredis |
-| [PHP](https://www.php.net/) | redis extension | https://github.com/phpredis/phpredis | https://github.com/phpredis/phpredis/blob/develop/README.md |
-| [PHP](https://www.php.net/) | predis library | https://github.com/predis/predis | https://github.com/predis/predis/wiki |
+| [PHP](https://www.php.net/) | PHPRedis extension | https://github.com/phpredis/phpredis | https://github.com/phpredis/phpredis/blob/develop/README.md |
```
Comment (dwdougherty) [inline content/develop/connect/clients/_index.md:42]: The README is titled "PhpRedis". Doesn't have to match, so I'll leave it to you.

```suggestion
| [PHP](https://www.php.net/) | PhpRedis extension | https://github.com/phpredis/phpredis | https://github.com/phpredis/phpredis/blob/develop/README.md |
```

File: content/develop/connect/clients/_index.md [hunk 2]
```diff
@@ -39,10 +39,11 @@ Redis does not support directly:
...
+| [PHP](https://www.php.net/) | Predis library | https://github.com/predis/predis | https://github.com/predis/predis/wiki |
 | [Ruby](https://www.ruby-lang.org/en/) | redis-rb | https://github.com/redis/redis-rb | https://rubydoc.info/gems/redis |
-| [Rust](https://www.rust-lang.org/) | redis-rs | https://github.com/redis-rs/redis-rs | https://docs.rs/redis/latest/redis/ | 
+| [Rust](https://www.rust-lang.org/) | redis-rs | https://github.com/redis-rs/redis-rs | https://docs.rs/redis/latest/redis/ |
+| [C++](https://en.wikipedia.org/wiki/C%2B%2B) | Boost.redis | https://github.com/boostorg/redis | https://www.boost.org/doc/libs/develop/libs/redis/doc/html/index.html |
```
Comment (dwdougherty) [inline content/develop/connect/clients/_index.md:46]: Similar comment to the last.

```suggestion
| [C++](https://en.wikipedia.org/wiki/C%2B%2B) | Boost.Redis | https://github.com/boostorg/redis | https://www.boost.org/doc/libs/develop/libs/redis/doc/html/index.html |
```

**Review by dwdougherty — APPROVED** [review]
Two things for you to consider. Approving so I'm not holding you up.

**Comment by andy-stark-redis** [comment]
@dwdougherty Good suggestions, thanks!

**Pull request merged**